openapi/data_repository_service.swagger.yaml

```diff
@@ -361,10 +377,22 @@ definitions:
           A list of full DRS identifier URI paths
           that may be used to obtain the object.
           These URIs may be external to this DRS instance.
+
+          If this ContentsObject describes a nested bundle, the drs_uri array may be
+          empty. A DRS server need not provide an object id or DRS URI for a nested bundle.
```

Comment:
I don't think this comment is needed, since I believe the `drs_uri` array is always optional.

Reply:
I will add the id field comment as you suggest. Here I was trying to capture the intent from our discussion.

I am curious what the rationale is for this to be optional in the non-nested case. It seems to say: there exists an object foo, but there is no way to access it.